soundbay/trainers.py

```diff
@@ -99,6 +100,9 @@ def train_epoch(self, epoch):
 
             # estimate and calc losses
             estimated_label = self.model(audio)
+            if estimated_label.shape[1] > 2: #Softmax in case of multiclass
```

Comment:
From reading https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html it seems like it should want to get un-normalized inputs (no softmax) for any number of classes.
In case I'm wrong here and the softmax is needed - need to add it also in the val loop.

Reply:
Right!
removed.